src/codeu/chat/client/commandline/Chat.java

```diff
@@ -107,6 +108,8 @@ public void invoke(Scanner args) {
         System.out.println("    Add a new user with the given name.");
         System.out.println("  u-sign-in <name>");
         System.out.println("    Sign in as the user with the given name.");
+        System.out.println("  version");
+        System.out.println("    Print the server version.");
         System.out.println("  exit");
         System.out.println("    Exit the program.");
```

Comment:
agreed... an enum would probably be the right construct here. (but no need to make that change).